src/modules/vtol_att_control/standard.cpp

```diff
@@ -535,7 +509,7 @@ void Standard::fill_actuator_outputs()
 
 		// zero outputs when inactive
 		_actuators_out_1->control[actuator_controls_s::INDEX_ROLL] = 0.0f;
-		_actuators_out_1->control[actuator_controls_s::INDEX_PITCH] = _params->fw_pitch_trim;
```

Comment:
Also here it should remain

Reply:
Since this trim is always applied (in FW and in MC mode) why not just adjusting mechanically the elevator and set this value to zero?